packages/components/accordion/examples/AccordionExample.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Accordion, Text } from '@contentful/f36-components';
+
+export default function IconButtonExample() {
+  return (
+    <Accordion>
```

Comment:
is there a way to control which items are opened programmatically? (controlled vs uncontrolled)